packages/templating/scanner_tests.js

```diff
@@ -30,13 +30,24 @@ Tinytest.add("templating - html scanner", function (test) {
   };
 
   // arguments are quoted strings like '"hello"'
-  var simpleTemplate = function (templateName, content) {
+  var simpleTemplate = function (templateName, templateAttributes, content) {
     // '"hello"' into '"Template.hello"'
     var viewName = templateName.slice(0, 1) + 'Template.' + templateName.slice(1);
 
-    return '\nTemplate.__checkName(' + templateName + ');\nTemplate[' + templateName +
+    var templateCode = '\nTemplate.__checkName(' + templateName + ');\nTemplate[' + templateName +
```

Comment:
I think we should replace this with a getter function. It feels like that will give us more flexibility for the future, and I think it would save a looooot of pain to make these attributes immutable.